Dockerfile.konflux

```diff
@@ -0,0 +1,68 @@
+
+#ARG BUILD_BASE=develop
+
+#go-1.21
+FROM --platform=$BUILDPLATFORM registry.redhat.io/ubi8/go-toolset:1.21@sha256:742ae6ec1aef3e7faae488c47695fb64964d342aefecf52d23bd9d5e6731d0b6 AS build
+
+USER root
+
+# needed for konflux as the previous stage is not used
+WORKDIR /opt/app
+COPY go.mod go.sum ./
+# Download dependencies before copying the source so they will be cached
+RUN go mod download
+
+# Copy the source
+COPY . ./
+
+ARG TARGETOS=amd64
+ARG TARGETARCH=linux
+
+# Build the binaries using native go compiler from BUILDPLATFORM but compiled output for TARGETPLATFORM
+# https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/
+RUN --mount=type=cache,target=/root/.cache/go-build \
+    --mount=type=cache,target=/go/pkg \
+    export GOOS=${TARGETOS:-linux} && \
+    export GOARCH=${TARGETARCH:-amd64} && \
+    go build -o puller model-serving-puller/main.go && \
+    go build -o triton-adapter model-mesh-triton-adapter/main.go && \
+    go build -o mlserver-adapter model-mesh-mlserver-adapter/main.go && \
+    go build -o ovms-adapter model-mesh-ovms-adapter/main.go && \
+    go build -o torchserve-adapter model-mesh-torchserve-adapter/main.go
+
+
+#Copy build assets to create the smallest final runtime image
+###############################################################################
+
+#ubi-minimal:latest
+FROM registry.redhat.io/ubi8/ubi-minimal@sha256:cf095e5668919ba1b4ace3888107684ad9d587b1830d3eb56973e6a54f456e67 as runtime
+
+ARG USER=2000
+
+USER root
+
+
```

Comment:
it is missing this: 
https://github.com/red-hat-data-services/modelmesh-runtime-adapter/blob/rhoai-2.16/Dockerfile.konflux#L48-L70